src/vuestic-theme/vuestic-components/vuestic-colors-presentation/VuesticColorsPresentation.demo.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="demo-container">
+    <div class="demo-container__item">
+      <div class="row">
+        <div class="col-md-12">
+          <h4>Colors</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-6" v-for="(mainThemeColor, index) in mainThemeColors" :key="index">
+          <vuestic-colors-presentation
+            :color="mainThemeColor.color"
+            :name="mainThemeColor.name"
+            :description="mainThemeColor.description"/>
+        </div>
+      </div>
+    </div>
+    <div class="demo-container__item">
+      <div class="row">
+        <div class="col-md-12">
+          <h4>Extra colors</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-6" v-for="(extraColor, index) in extraColors" :key="index">
+          <vuestic-colors-presentation
```

Comment:
@asvae , @nastassiadanilova , let's decide on the naming here. Maybe singular `vuestic-color-presentation` since we pass in single color?

Thanks!

Reply:
I agree with you. I have written as it be mentioned in task. It's good idea to rename